supertrader/rl/agent.py

```
@@ -0,0 +1,20 @@
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
```

Flake8 (7.2.0)

[error] 'stable_baselines3.common.vec_env.DummyVecEnv' imported but unused

(F401)